src/Form/Layout.php

```diff
@@ -199,22 +199,25 @@ public function recursiveRender()
                 $label = $this->label;
                 $this->label = null;
             } elseif ($this->label || $this->inline) {
-                $template = $field_no_label;
+                $template = $noLabelInput;
                 $el->placeholder = $label;
             }
 
             // Fields get extra pampering
             $template->setHTML('Input', $el->getHTML());
             $template->trySet('label', $label);
             $template->trySet('label_for', $el->id . '_input');
-            $template->set('field_class', $el->getControlClass());
+            $template->set('control_class', $el->getControlClass());
+
+            // backward compatibility - will be removed in dec-2020
+            $template->trySet('field_class', $el->getControlClass());
```

Comment:
BC without deprecation warning is good as a release will be made, but users will not notice until the support is dropped

I this is is not used that much/on so many places in custom apps, so I would remove it from templates and throw an exception here if a value was/is set (but with note to remove that if/exception later)